src/Integration/Persistence/BoundSonarQubeProject.cs

```diff
@@ -38,6 +39,8 @@ public BoundSonarQubeProject(Uri serverUri, string projectKey, ICredentials cred
 
         public string ProjectKey { get; set; }
 
+        public Dictionary<LanguageGroup, ApplicableQualityProfile> Profiles { get; set; }
```

Comment:
Decided on a project
1) Reads nice as json (see example in this PR)
2) We need to compare a profile for language vs. a profile for the same language, dictionary provides that aggregation by definition.